monitoring/bin/deploy_monitoring_cluster.sh

```diff
@@ -229,6 +220,24 @@ else
   log_debug "PrometheusRule $MON_NS/v4m-kubernetes-apps does not exist"
 fi
 
+# Elasticsearch Datasource for Grafana
+# Moved down to make sure Grafana pods exist
+ELASTICSEARCH_DATASOURCE="${ELASTICSEARCH_DATASOURCE:-false}"
+if [ "$ELASTICSEARCH_DATASOURCE" == "true" ]; then
+  set +e
+  monitoring/bin/create_elasticsearch_datasource.sh
+
+  if (( $? == 1 )); then
+    log_warn "Unable to configure the Elasticsearch data source at this time."
+    log_warn "Please address the errors and re-run the follow command:"
+    log_warn "monitoring/bin/create_elasticsearch_datasource.sh"
+  fi
+  set -e
+else
+  log_debug "ELASTICSEARCH_DATASOURCE not set"
+  log_debug "Skipping creation of Elasticsearch datasource for Grafana"
```

Comment:
Let's refer to the datasource as "Viya Logs datasource for Grafana" in this script (and eventually use that elsewhere as well).  That eliminates the (soon to be inaccurate) reference to Elasticsearch and is probably more meaningful to users anyway.

Reply:
While I didn't add this line of code, I disagree with changing the wording of this particular message.

The environment variable is ELASTICSEARCH_DATASOURCE.  The script is create_ELASTICSEARCH_datasource.  When we fully convert over to OpenSearch, we can discuss how we want to approach the two data source system; however, a new environment variable and script will be needed and we can adjust/replace the message then.

Reply:
We have several environment variables that reference Elasticsearch or Kibana in some way.  We're going to have to live with that for some period of time.  We will (probably) eventually move to new names and support both during some transition phase.  We're trying to address references to those product names in messages and file names as we move to OpenSearch.  This is a DEBUG level message, so I suppose changing this now isn't critical.